src/index.php

```diff
@@ -0,0 +1,71 @@
+<?php require($_SERVER['DOCUMENT_ROOT'] . "/db_connect.php");
+// $id = htmlspecialchars($_GET["id"]);
+// jsを書き込んだとしても、ただの文字として扱ってほしい
+// prefectures
+if (isset($_GET['id'])) {
+  $id = $_GET['id'];
+  $prefectures_value =  "SELECT * FROM prefectures WHERE id = $id";
+  $prefectures = $db->query($prefectures_value)->fetchAll(PDO::FETCH_ASSOC | PDO::FETCH_UNIQUE);
+}
+echo ($prefectures[$id]["name"]);
+// choices
+if (isset($_GET['id'])) {
+  $id = $_GET['id'];
```

Comment:
なんのidかわかりやすい変数名の方が良さそうです。
```suggestion
  $prefectureId = $_GET['id'];
```